proto/spire/api/server/entry/v1/entry.proto

```diff
@@ -46,6 +46,21 @@ service Entry {
     // The caller must present an active agent X509-SVID. See the Agent
     // AttestAgent/RenewAgent RPCs.
     rpc GetAuthorizedEntries(GetAuthorizedEntriesRequest) returns (GetAuthorizedEntriesResponse);
+
+
+    // Syncs authorized entries down the caller. The caller controls which
+    // entries the server sends down full details for. The flow is as follows:
+    // 1. Caller opens up sync stream
+    // 2. Server determines authorized entries for caller:
+    //    - If there are less entries than a server-determined page size, go to (5).
```

Comment:
it is going to 5? or 4?
this means that we get full if we return entries < pagesize?

Reply:
To (5). If there aren't many entries, it's not worth trying to do the "diff". The server just sends the full non-sparse set to the agent.
The outcome is that there is no noticeable difference from a network standpoint for existing deployments whose authorized entry sets are smaller than the page size.